src/Veldrid.StartupUtilities/VeldridStartup.cs

```diff
@@ -97,8 +96,10 @@ public static GraphicsDevice CreateGraphicsDevice(Sdl2Window window, GraphicsBac
             {
                 case GraphicsBackend.Direct3D11:
                     return CreateDefaultD3D11GraphicsDevice(options, window);
+#if GFXAPI_VULKAN_ENABLED
```

Comment:
This should throw a VeldridException with some message like "Vulkan support has not been included in this configuration of Veldrid." ATM it will go through the `default` case below with a generic message.